source/GameData.cpp

```diff
@@ -1045,7 +1045,8 @@ void GameData::PrintShipTable()
 		<< "mass" << '\t' << "crew" << '\t' << "cargo" << '\t' << "bunks" << '\t'
 		<< "fuel" << '\t' << "outfit" << '\t' << "weapon" << '\t' << "engine" << '\t'
 		<< "speed" << '\t' << "accel" << '\t' << "turn" << '\t'
-		<< "e_gen" << '\t' << "e_use" << '\t' << "h_gen" << '\t' << "h_max" << '\n';
+		<< "e_gen" << '\t' << "e_use" << '\t' << "h_gen" << '\t' << "h_max" << '\t'
+		<< "guns" << '\t' << "turrets" << '\n';
```

Comment:
Overall change looks ok to me. One detail, I think "gun_ports" and "turrent_mounts" are more appropriate as header entries, since you are not printing the actual guns and turrets, but only the places to install them in.

Reply:
Done, went with `gun_mnts` / `turret_mnts` for brevity.